pints/_mcmc/_results.py

```diff
@@ -0,0 +1,204 @@
+#
+# MCMC results method
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017-2019, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pints
+import numpy as np
+from tabulate import tabulate
+
+
+class MCMCResults(object):
```

Comment:
I'm wondering if there's a better name, like MCMCResultAnalysis, or MCMCResultSummary or something

Reply:
MCMCSummary, MCMCDiagnosis, MCMCAnalysis ?

Reply:
Yep, good idea! Have changed it to MCMCSummary.